packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -727,36 +736,35 @@ export class SynapseTable extends React.Component<
                     </span>
                   )}
                   {isSortableColumn(column.columnType) && (
-                    <span
-                      role="button"
-                      aria-label="sort"
-                      aria-selected={isSelected}
-                      tabIndex={0}
-                      onKeyPress={this.handleColumnSortPress({
-                        index,
-                        name: column.name,
-                      })}
-                      onClick={this.handleColumnSortPress({
-                        index,
-                        name: column.name,
-                      })}
+                    <Tooltip
+                      title={`Sort ${getColumnDisplayName(column.name)}`}
+                      placement={'top'}
                     >
-                      <IconSvg
-                        icon={ICON_STATE[columnIndex]}
-                        wrap={false}
-                        sx={{
-                          color: isSelected
-                            ? 'primary.contrastText'
-                            : 'primary.main',
-                          backgroundColor: isSelected ? 'primary.main' : 'none',
-
-                          '&:hover': {
-                            color: 'primary.contrastText',
-                            backgroundColor: 'primary.600',
-                          },
-                        }}
-                      />
-                    </span>
+                      <IconButton
+                        role="button"
+                        aria-label="sort"
+                        size={'small'}
+                        aria-selected={isSelected}
+                        tabIndex={0}
+                        onKeyPress={this.handleColumnSortPress({
+                          index,
+                          name: column.name,
+                        })}
+                        onClick={this.handleColumnSortPress({
+                          index,
+                          name: column.name,
+                        })}
+                      >
+                        <IconSvg
+                          icon={ICON_STATE[columnIndex]}
+                          wrap={false}
+                          sx={{
+                            color: isSelected ? 'primary.main' : 'grey.700',
+                            backgroundColor: 'none',
+                          }}
+                        />
+                      </IconButton>
+                    </Tooltip>
```

Comment:
Use `IconButton` for the sort indicator. Change the styling to match the design (making some assumptions about how the "active" state should appear)

Reply:
Example of active sort, active filter: 
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/97fdf953-5610-41e1-a6f1-5112ee93647c)